cmd/run/run_test.go

```diff
@@ -46,6 +37,16 @@ import (
 	"google.golang.org/grpc/credentials/insecure"
 	healthv1pb "google.golang.org/grpc/health/grpc_health_v1"
 	"google.golang.org/protobuf/encoding/protojson"
+
+	"github.com/openfga/openfga/cmd"
+	"github.com/openfga/openfga/cmd/util"
+	"github.com/openfga/openfga/internal/mocks"
+	serverconfig "github.com/openfga/openfga/internal/server/config"
+	"github.com/openfga/openfga/pkg/logger"
+	serverErrors "github.com/openfga/openfga/pkg/server/errors"
```

Comment:
@sergiught thank you! out of curiosity, do you know of a neat way to enforce aliases to be lowercase? I added importas specifically for aliased imports on `github.com/openfga/api/proto/openfga/v1`.

Reply:
@jpadilla My pleasure! 

> out of curiosity, do you know of a neat way to enforce aliases to be lowercase? 
As far as I know this isn't supported out of the box with golangci-lint. However we could perhaps introduce https://pre-commit.com/ and add a custom script that would do that for us. I can research this further at a later point in time if that's alright with you. 

PS. I can't seem to be able to merge this PR even with the approvals. As the base branch restricts merging to authorized users only.